backend/runner/approval/runner.go

```diff
@@ -158,7 +158,7 @@ func (r *Runner) findApprovalTemplateForIssue(ctx context.Context, issue *store.
 			err = errors.Wrapf(err, "failed to get approval template, riskLevel: %v", riskLevel)
 		}
 
-		return approvalTemplate, false, err
+		return approvalTemplate, true, err
```

Comment:
Add a comment about what this bool means?

Reply:
line 138 does it.